data/misc/coco/coco_sample.json

```diff
@@ -0,0 +1,120 @@
+{
+    "info": {
+        "year": "2020",
+        "version": "1",
+        "description": "",
+        "contributor": "",
+        "url": "https://url",
+        "date_created": "2020-01-01T00:00:00+00:00"
+    },
+    "licenses": [
+        {
+            "id": 1,
+            "url": "",
+            "name": "name"
+        }
+    ],
+    "categories": [
+        {
+            "id": 0,
+            "name": "bottle",
+            "supercategory": "none"
+        },
+        {
+            "id": 1,
+            "name": "can",
+            "supercategory": "cells"
+        },
+        {
+            "id": 2,
+            "name": "carton",
+            "supercategory": "cells"
+        }
+    ],
+    "images": [
+        {
+            "id": 0,
+            "license": 1,
+            "file_name": "image0.jpg",
+            "height": 480,
+            "width": 640,
+            "date_captured": "2020-01-01T00:00:00+00:00",
+			"coco_url": "http://image0.jpg"
```

Comment:
this formatting is weird

Reply:
Yes. Looks like this extra tab is coming from github, it looks good in a text editor.

Reply:
[nit] although its hard to see, I think there actually is a tab instead of spaces here.

Reply:
Thanks. I deleted the file and also ran black on the xml string.